### ISSUE TITLE
Error Parsing xhtml with backticks code

### ISSUE THREAD

**Issue by unstableunicorn** [sphinx-contrib/confluencebuilder#111]
Everything was previously working ok until I tried to push some changes to Confluence today and got the following:
```
publishing... [  7%] index                                                                                                                                                                                          
sphinxcontrib.confluencebuilder error:
---
An unsupported Confluence API call has been made.

REQ: PUT
RSP: 400
URL: https://<domain>.atlassian.net/wiki/rest/api/
API: content
MSG: Error parsing xhtml: Unexpected close tag </code>; expected </span>.
 at [row,col {unknown-source}]: [192,160]
---

Makefile:20: recipe for target 'confluence' failed
make: *** [confluence] Error 2
```

It seems it has an issue formatting the following line:

```
   some text ``-h`` or ``--help`` command.
```
I though I already had this in my docs but was around the other way.
Changing to:
```
   some text ``--help`` or ``-h`` command.
```

And everything worked.
Changing back and it failed again.

**Comment by jdknight** [comment]
This issue is (or should be) resolved on `master` will be made available next release (see #95). For more information on why this happens, you can also consult the PR #72 and the issue #62.

While I'm going to mark this issue as a duplicate, I will personally not close this issue until the next release has been published to PyPI (so others who are tracking this issue still know it's pending an official release).

@unstableunicorn, if you're stuck, you could try the [bleeding edge of the source](http://sphinxcontrib-confluencebuilder.readthedocs.io/en/latest/install.html#master); otherwise wait until the next release has been completed (hopefully soon).

**Comment by unstableunicorn** [comment]
Great you got it solved quickly!
Swapping the text has solved it for me at the moment so no need to use master.
I have no real need to upgrade until issue #51 is completed. That issue is a big one as I use a lot of autodoc and would make it a lot nicer to view.
